src/lib/mainEvent.js

```diff
@@ -147,6 +147,17 @@ mainEvent.on("user:unlock", (fields, resolve) => {
   }).then(resolve);
 });
 
+// installer_version
+mainEvent.on("user:installer_version", (version, resolve) => {
+  log.warn(`Incompatible UBports Installer version! Required: ${version}`);
+  prompt({
+    title: "Incompatible installer version",
```

Comment:
Is this codecov warning a false positive? How could these lines not be covered but all others?

Reply:
the registration of the eventListener (calling `mainEvent.on()`) is executed, the actual listener isn't. The object literal passed to `prompt()` is evaluated statically, but never executed. The `.then()` function in never called thusly. CodeCov is correct, but the warning doesn't matter. we're not properly testing `mainEvent.js` yet.

Reply:
LOL did not see your comment in the first place :P